examples/step-57/doc/intro.dox

```diff
@@ -0,0 +1,319 @@
+\cdot \nabla)\textbf{u}^k -\nabla p^k$ and $\textbf{u}^k$ and $p^k$ are the solutions from the
+previous iteration. In this linear system, the only unknowns are the
+update terms $\delta \textbf{u}^{k}$ and $\delta p^{k}$, and we can use a similar strategy
+to the one used in step-22. The weak form is
+derived like it is done in step-22.
+
+Now, Newton's iteration can be used to solve for it:
+
+<ol>
+  <li> Initialization: Initial guess $u_0$ and $p_0$, tolerance $\tau$;
+  <li> Linear solve to compute update term $\delta\textbf{u}^{k}$ and $\delta p^k$;
+  <li> Update the approximation: $\textbf{u}^{k+1} = \textbf{u}^{k} + \delta\textbf{u}^{k}$ and $p^{k+1} = p^{k} + \delta p^{k}$;
+  <li> Check residual norm: $E^{k+1} = \|F(\mathbf{u}^{k+1}, p^{k+1})\|$. 
+       If $E^{k+1} \leq \tau$, STOP. 
+       If $E^{k+1} > \tau$, back to step 2. 
+</ol>
+
+<h3> Finding an Initial Guess </h3>
+
+Getting Newton's method to converge, the initial guess needs to be close
+enough to the solution, so it is crucial to find a good starting value.
+
+When the viscosity $\nu$ is large, a good initial guess can be obtained
+by solving the Stokes equation with viscosity $\nu$. While problem dependent,
+this works for $\nu \geq 1/400$ for the test problem considered here.
+
+However, the convective term $(\mathbf{u}\cdot\nabla)\mathbf{u}$ will be
+dominant if the viscosity is small, like 1/7500 in test case 2.  In this
+situation, we use a continuation method to set up a series of auxiliary NSE with
+viscosity approaching the one in the target NSE. Correspondingly, we create a
+sequence $\{\nu_{i}\}$ with $\nu_{n}= \nu$, and accept that the solutions to
+two NSE with viscosity $\nu_{i}$ and $\nu_{i+1}$ are close if $|\nu_{i} -
+\nu_{i+1}|$ is small.  Then we use the solution to the NSE with viscosity
+$\nu_{i}$ as the initial guess of the NSE with $\nu_{i+1}$. This can be thought of
+as a staircase from Stokes equations to the NSE we want to solve.
+
+That is, we first solve a Stokes problem
+
+@f{eqnarray*}
+- \nu_{1} \Delta\textbf{u} + \nabla p &=& \textbf{f}\\
+- \nabla \cdot \textbf{u} &=& 0
+@f}
+
+to get the initial guess for
+
+@f{eqnarray*}
+- \nu_{1} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f},\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+Here $\nu_{1}$ is relatively large so that the solution to Stokes problem with viscosity $\nu_{1}$
+can be used as an initial guess for NSE in Newton's iteration. 
+
+Then the solution to
+
+@f{eqnarray*}
+- \nu_{i} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f},\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+acts as the initial guess for
+
+@f{eqnarray*}
+- \nu_{i+1} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f},\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+We repeat this process until we can use the solution to the NSE with viscosity $\nu_{n-1}$ as the 
+initial guess for the NSE with viscosity $\nu$.
```

Comment:
But you say "We repeat this process until we can use the solution". This suggests that there is a criterion that can be evaluated to test whether the solution can be used. If I understand your comment correctly, then there is no such criterion other than trial and error. Is this correct?

If so, maybe write this into intro.dox: "This process is repeated with a sequence of viscosities, $\{\nu_n\}$ that is determined experimentally so that the final solution can used as a starting guess for the Newton iteration."